src/main/java/teammates/storage/sqlentity/UsageStatistics.java

```diff
@@ -0,0 +1,157 @@
+package teammates.storage.sqlentity;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+/**
+ * Represents a system usage statistics for a specified period of time.
+ *
+ * <p>Note that "system usage" here is defined as user-facing usages, such as number of entities created
+ * and number of actions, as opposed to system resources such as hardware and network.
+ */
+@Entity
+@Table(name = "UsageStatistics")
+public class UsageStatistics extends BaseEntity {
+    @Id
+    private String id;
```

Comment:
Let's use serial ids instead, no real reason to generate our own string based IDs now.